CSSTooltipPseudo/explainer.md

```diff
@@ -0,0 +1,897 @@
+images/out-of-bounds-firefox-windows.png)
+
+
+#### Webkit
+
+In Webkit, I was *not* able to find a limit for built-in tooltip length for
+which an elipses was added, like it does in Chromium. However, tooltips
+are also able to escape the bounds of the window in Webkit. In this case,
+the tooltip appears to have uneven padding. For example:
+
+![Screenshot of a long tooltip in Webkit on Mac](
+images/out-of-bounds-webkit-mac.png)
+
+_________________________
+
+It is clear that the tooltips provided by the top browser engines are
+inconsistent in styling between each other and across OS, and there are
+divergences in positioning and some behaviors with longer tooltip text.
+
+This inconsistency and lack of customizability leads authors to find
+alternatives that often require more work and maintenance.
+
+## The `::tooltip` pseudo element
+
+Many have proposed a pseudo for styling browser tooltips over the years,
+with the [first proposal](
+https://lists.w3.org/Archives/Public/www-style/2000Apr/0014.html)
+originating around April 2000, and the most [recent proposal](
+https://github.com/w3c/csswg-drafts/issues/8930#issuecomment-1581848840)
+from Lea Verou in 2023. 
+
+The proposal in this document takes close inspiration from these, by
+introducing a new pseudo element, called `::tooltip`, that can be
+used by authors to style the browser tooltip that is displayed when
+the `title` attribute is set on an HTML element.
+
+The `::tooltip` pseudo element will have default UA styles that will
+be required to be applied when an author adjusts tooltip styles
+within a `::tooltip` pseudo element selector.
+
+The idea behind this proposal is that authors would be able to
+create simple text-based tooltips, and adjust the styles of color
+properties, shadowing, border properties, font properties, etc
+to better match the look and feel of their site, leaving
+positioning, input handling, and accessibility up to the browser.
+
+### How to trigger `::tooltip` default styles?
+
+Currently, browser default tooltips are triggered by the `title`
+attribute being set, or by the presence of the `<title>` element for
+SVG elements.
+
+We could have the default `::tooltip` apply in all cases by default,
+but this may be a breaking change for some browsers. Instead, the UA
+should be able to style tooltips in their own chosen way, unless
+`::tooltip` is triggered in some way.
+
+Authors may also want to trigger this path without triggering any
+styles beyond those that are default. As such, there requires
+some mechanism for authors to trigger the new `::tooltip`-based
+default.
+
+An initial suggestion would be to add a new value to [`appearance`](
+https://drafts.csswg.org/css-ui/#propdef-appearance), similar to
+what was done for [customizable select elements](
+https://developer.mozilla.org/en-US/docs/Learn_web_development/Extensions/Forms/Customizable_select),
+called `base-tooltip`, that an author can use within the `::tooltip`
```

Comment:
Why `base-tooltip` rather than `base` like we have in [CSS Forms 1](https://www.w3.org/TR/css-forms-1)?

Reply:
I was looking at the current definition of `appearance` which was why I added a new value for this, but that would make sense to me. Updated to use `base` and updated all example code using `base-tooltip` to `base`